Providers/PythonProvider.cpp

```diff
@@ -198,7 +197,7 @@ namespace scx
 
 
 /*static*/ unsigned char const PythonProvider::MI_NULL_FLAG = 64;
-
+/*static*/ MI_Boolean PythonProvider::CHILD_SIGNAL_REGISTERED = MI_FALSE;
```

Comment:
Let's use ChildSignalRegister instead of CHILD_SIGNAL_REGISTERED... or are we using all caps for statics elsewhere?

Reply:
we are using caps for static member variables.

---
In reply to: [143595148](https://github.com/Microsoft/PowerShell-DSC-for-Linux/pull/331#discussion_r143595148) [](ancestors = 143595148)